src/main/java/com/exasol/validation/GitHubPlatformValidator.java

```diff
@@ -70,19 +65,23 @@ protected void validateGitHubTickets(final ReleaseLetter changesFile) {
         final List<String> wrongTickets = collectWrongTickets(changesFile);
         if (!wrongTickets.isEmpty()) {
             reportWrongTickets(changesFile.getFileName(), wrongTickets);
+        } else {
+            this.validationReport.addSuccessfulValidation("Mentioned GitHub tickets.");
         }
     }
 
     private void reportWrongTickets(final String fileName, final List<String> wrongTickets) {
         final String wrongTicketsString = String.join(", ", wrongTickets);
         if (this.branchContent.isDefaultBranch()) {
-            throw new IllegalStateException(
-                    "E-RR-VAL-2: Some of the mentioned GitHub issues are not closed or do not exists: "
-                            + wrongTicketsString + ", Please, check the issues numbers in your '" + fileName
-                            + "' one more time.");
+            this.validationReport.addFailedValidations("E-RR-VAL-2",
+                    "Some of the mentioned GitHub issues are not closed or do not exists: " + wrongTicketsString
+                            + ", Please, check the issues numbers in your '" + fileName + "' one more time.");
         } else {
-            LOGGER.warning("Don't forget to close the tickets mentioned in the '" + fileName
-                    + "' file before you release: " + wrongTicketsString);
+            final String warningMessage = "Don't forget to close the tickets mentioned in the '" + fileName
+                    + "' file before you release: " + wrongTicketsString + ".";
+            this.validationReport
+                    .addSuccessfulValidation("Skipping mentioned GitHub tickets validation. " + warningMessage);
+            LOGGER.warning(warningMessage);
```

Comment:
The unified error handling also includes codes for warnings (starting with `W-`). I think you could one here.